parser/helper/helpers.go

```diff
@@ -149,6 +207,12 @@ func (h *Helper) GetActorNameFromAddress(add address.Address, height int64, key
 	if add == address.Undef {
 		return cid.Undef, "", errors.New("address is undefined")
 	}
+	// The f090 address was a multisig actor until V23 where it was converted to an account actor
+	// https://github.com/filecoin-project/lotus/releases/tag/v1.28.1
+	// https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0085.md
+	if ok, cid, actorName := h.isKeylessAccountActor(add, height); ok {
+		return cid, actorName, nil
+	}
```

Comment:
With this one.... maybe we can extract and reuse? to avoid possible mismatches between the two in the future?

Reply:
It would be a single entry point for actor cid and type, as we do on GetActorNameFromCid, which consumes the special cases for actors